python/ray/serve/_private/deployment_state.py

```diff
@@ -1771,6 +1775,9 @@ def deploy(self, deployment_name: str, deployment_info: DeploymentInfo) -> bool:
             self._deployment_states[deployment_name] = self._create_deployment_state(
                 deployment_name
             )
+            record_extra_usage_tag(
+                TagKey.SERVE_NUM_DEPLOYMENTS, str(len(self._deployment_states))
```

Comment:
Can we have a function to accept the int instead of converting to str? 

(Not python expert, but it looks like inefficient)